docs/core/run-time-config/garbage-collector.md

```diff
@@ -0,0 +1,204 @@
+- Applies to server garbage collection (GC) only.
+- Default: Affinitize garbage collection threads with processors (`false`).
+
+| | Setting name | Values | Version introduced |
+| - | - | - | - |
+| **runtimeconfig.json** | `System.GC.NoAffinitize` | `false` - affinitize<br/>`true` - don't affinitize | .NET Core 3.0 |
+| **Environment variable** | `COMPlus_GCNoAffinitize` | 0 - affinitize<br/>1 - don't affinitize | .NET Core 3.0 |
+| **app.config for .NET Framework** | [GCNoAffinitize](../../framework/configure-apps/file-schema/runtime/gcnoaffinitize-element.md) | `false` - affinitize<br/>`true` - don't affinitize | 4.6.2 |
+
+### System.GC.HeapHardLimit/COMPlus_GCHeapHardLimit
+
+- Specifies the maximum commit size, in bytes, for the GC heap.
+- The value can range from 0 to 18,446,744,073,709,551,615.
+
+| | Setting name | Values | Version introduced |
+| - | - | - | - |
+| **runtimeconfig.json** | `System.GC.HeapHardLimit` | *decimal value* | .NET Core 3.0 |
+| **Environment variable** | `COMPlus_GCHeapHardLimit` | *decimal value* | .NET Core 3.0 |
+
+### System.GC.HeapHardLimitPercent/COMPlus_GCHeapHardLimitPercent
+
+- Specifies the GC heap usage as a percentage of the total memory.
+- Example value: 25
+
+| | Setting name | Values | Version introduced |
+| - | - | - | - |
+| **runtimeconfig.json** | `System.GC.HeapHardLimitPercent` | *percentage* | .NET Core 3.0 |
+| **Environment variable** | `COMPlus_GCHeapHardLimitPercent` | *percentage* | .NET Core 3.0 |
+
+### System.GC.RetainVM/COMPlus_GCRetainVM
+
+- Configures whether segments that should be deleted are put on a standby list for future use or are released back to the operating system (OS).
+- Default: Release segments back to the operating system (`false`).
+
+| | Setting name | Values | Version introduced |
+| - | - | - | - |
+| **runtimeconfig.json** | `System.GC.RetainVM` | `false` - release to OS<br/>`true` - put on standby| .NET Core 1.0 |
+| **Environment variable** | `COMPlus_GCRetainVM` | 0 - release to OS<br/>1 - put on standby | .NET Core 1.0 |
+
+## Large pages
+
+### COMPlus_GCLargePages
+
+- Specifies whether large pages should be used when a heap hard limit is set.
+- Default: Disabled (0).
+- This is an experimental setting.
+
+| | Setting name | Values | Version introduced |
+| - | - | - | - |
+| **runtimeconfig.json** | N/A | N/A | N/A |
+| **Environment variable** | `COMPlus_GCLargePages` | 0 - disabled<br/>1 - enabled | .NET Core 1.0 |
+| **app.config for .NET Framework** | N/A | N/A | N/A |
+
+## Large objects
+
+### COMPlus_gcAllowVeryLargeObjects
+
+- Configures garbage collector support on 64-bit platforms for arrays that are greater than 2 gigabytes (GB) in total size.
+- Default: Enabled (1).
+- This option may become obsolete in a future version of .NET.
+
+| | Setting name | Values | Version introduced |
```

Comment:
this is also available on desktop: https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/gcallowverylargeobjects-element